server/controllers/finance/debtors/index.js

```diff
@@ -266,7 +266,8 @@ function financialPatient(debtorUuid) {
       LEFT JOIN project ON invoice.project_id = project.id 
       WHERE general_ledger.entity_uuid = ? AND general_ledger.entity_type = 'D'
     ) as transaction
-    GROUP BY transaction.trans_id;`;
+    GROUP BY transaction.trans_id 
+    ORDER BY transaction.trans_date ASC;`;
```

Comment:
I notice that the user interface in the patient billing zone display some data when there are not any invoices for the patient 
![last invoice](https://cloud.githubusercontent.com/assets/5445251/19351975/cc295204-9112-11e6-87e6-ccc382f3ae47.png)

Could you hide these informations when there are not invoices for the patient, and display a meaningful message such as `No invoices found for the patient`  well centered in the zone for example.